magicctapipe/scripts/lst1_magic_real/data/config.yaml

```diff
@@ -43,41 +73,82 @@ event_coincidence:
     
 
 stereo_reco: 
-    subarray: '/fefs/home/yoshiki.ohtani/software/magic-cta-pipe/lst1_magic/data/subarray_lst1_magic.pkl'
-    quality_cuts: (intensity > 50) & (width > 0)
+    subarray: '/home/yoshiki.ohtani/software/magic-cta-pipe/magicctapipe/scripts/lst1_magic_real/data/subarray_lst1_magic.pickle'
```

Comment:
I think that is better to save the subarray information together with the DL1 file(s) produced by the event coincidence script. If you have the SubarrayDescription object (as saved in the pickle file), you can define it in the event coincidence script and use the SubarrayDescription.to_hdf method to write it into the HDF5 file (if I understood correctly, saving the subarray in the HDF5 file should be done before saving anything else). 
This would avoid having to specify the path to the pickle file, and instead reading everything from the file at once.

Reply:
Hm, OK, I think I can define the subarray object inside the stereo reconstruction script. I'm implementing this solution now.

Reply:
By the way saving the subarray after saving others, for example event dl1 parameters works without any problems. Probably it was updated?